### ISSUE TITLE
"didn't understand the very end of your file" errors propagate upwards from imports

### ISSUE THREAD

**Issue by elfprince13** [brownplt/pyret-lang#395]
I have two files, let's call them "code.arr" and "tests.arr":

code.arr contains:

```
fun x(y):
    y
```

tests.arr contains:

```
import x from my-gdrive("code.arr")
check "identity":
    x(1) is 1
end
```

Running tests.arr does not provide any location information for the error originating in code.arr.

**Comment by jpolitz** [comment]
@jack – this is still an issue, thoughts on fixes?

![image](https://user-images.githubusercontent.com/119891/28503126-ca40be2a-6fbd-11e7-8da7-3085731e1f3c.png)

**Comment by jswrenn** [comment]
Thanks for tagging me; I hadn't seen this bug report. As noted in brownplt/code.pyret.org#153, challenges with projects spanning multiple files were the top reported frustration of students.

Dream fix: introduce tabbed editing into CPO.
Pragmatic fix: as with #940, we can just throw a `cmcode` snippet after the prose of this message.
